.circleci/custom_scripts/start_db_container.sh

```diff
@@ -8,4 +8,4 @@ docker run \
   -e POSTGRES_DB=mtp_api \
   circleci/postgres:10.10-alpine
 
-sleep 10
+ps_isready --timeout 10 --host postgres
```

Comment:
this won't work as it's outside the container with the `circleci/postgres:10.10-alpine` image

Reply:
as I misspelled the binary name I'm pretty sure this wouldn't be in that image :P 

but more seriously, I see what you mean, I give it a quick try but I'll drop the commit 👍

Reply:
(attempt/commit removed)